src/utils/routes.tsx

```diff
@@ -51,42 +53,48 @@ export const routes = [
     subtitle: 'Pasirinkite žvejybos vietą',
     slug: slugs.fishingLocation,
     iconName: IconName.home,
-    component: <Fishing />,
-    regExp: new RegExp('^/zvejyba$'),
+    component: <CurrentFishing />,
+    regExp: new RegExp('^/zvejyba/vieta$'),
   },
   {
     title: 'Mano žvejyba',
     subtitle: 'Pasirinkite žvejybos veiksmą',
-    slug: slugs.fishing(':fishingId'),
-    component: <Fishing />,
-    regExp: new RegExp('^/zvejyba/[0-9]+$'),
+    slug: slugs.fishingCurrent,
+    component: <CurrentFishing />,
+    regExp: new RegExp('^/zvejyba/mano$'),
   },
   {
-    slug: slugs.fishingTools(':fishingId'),
-    component: <FishingTools />,
+    slug: slugs.fishingTools,
+    component: <CurrentFishingTools />,
     back: true,
   },
-
   {
-    slug: slugs.fishingToolConnect(':fishingId', ':toolId'),
-    component: <Fishing />,
-    regExp: new RegExp('^/zvejyba/[0-9]+/irankiai[0-9]+/irankiu_jungimas$'),
+    slug: slugs.fishingToolConnect(':toolId'),
+    component: <CurrentFishing />,
+    regExp: new RegExp('^/zvejyba/mano/irankiai[0-9]+/irankiu_jungimas$'),
```

Comment:
maybe it would be enough react router built in hook/function usematch/matchpath instead of custom regex ?

Reply:
Sitas budas irgi veikia, tai jei bus noro galima bus perrasyti.